TestCases/compliance-level-3/0004-lending/0004-lending.dmn

```diff
@@ -1284,7 +1284,7 @@
                     </inputExpression>
                 </input>
                 <input id="_f692dbba-dc13-4076-a5ba-275041c0b944" label="CreditScore">
-                    <inputExpression typeRef="string">
+                    <inputExpression typeRef="number">
```

Comment:
Analogous comment as above, also for parameter "Post-bureauRiskCategory" used in the decision table